### PR TITLE
Examinable Armor

### PR DESCRIPTION
**Pull request by Geevies**
* Added the ability to examine items and see their armor values.

image

image

### CODE REVIEW

**Comment by DreamySkrell** [comment]
Idk, I don't think in-game characters need to see this. 

I think it'd be nicer if it wasn't actual percentage values, but instead "this item has heavy ballistic plating" or "this item has no armor at all". Just some text descriptions.

Similarly to how when examining people you don't see "they are 192 cm tall" but instead "they look about average height" or "they look tall for a tajara".

**Comment by DreamySkrell** [comment]
As discussed on discord, another suggestion would be to clearly mark this as OOC information. 

Cause it'd be weird to IC see characters discuss this like "damn this armor vest is shit, it got only 50% bullet armor".

**Comment by Geevies** [comment]
image

**Comment by Geevies** [comment]
!review

**Review by DreamySkrell — APPROVED** [review]

File: tgui/packages/tgui/interfaces/ArmorValues.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section>
+          <Box>
+            THE STATISTICS BELOW IS OUT OF CHARACTER INFO, YOU CAN USE THIS TO
+            REFERENCE ARMOR VALUES, BUT DO NOT STATE THE PERCENTAGES IN
+            CHARACTER
+          </Box>
```
Comment (NonQueueingMatt) [inline tgui/packages/tgui/interfaces/ArmorValues.tsx:20]: NoticeBox this and make it not caps. Also:
`the statistics below are* out of character info`

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by Geevies** [comment]
image

**Comment by Geevies** [comment]
!review

File: tgui/packages/tgui/interfaces/ArmorValues.tsx (hunk 1 above)
Comment (DreamySkrell) [reply tgui/packages/tgui/interfaces/ArmorValues.tsx]: fixed

**Review by DreamySkrell — APPROVED** [review]

File: tgui/packages/tgui/interfaces/ArmorValues.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const { act, data } = useBackend<ArmorValuesData>(context);
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section>
+          <NoticeBox>
+            The statistics below are out of character info, you can use this to
+            reference armor values, but do not state the percentages in
+            character
```
Comment (NonQueueingMatt) [inline tgui/packages/tgui/interfaces/ArmorValues.tsx:19]:
```suggestion
            character.
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**